src/libs/actions/Plaid.ts

```diff
@@ -6,13 +6,17 @@ import * as PlaidDataProps from '../../pages/ReimbursementAccount/plaidDataPropT
 
 /**
  * Gets the Plaid Link token used to initialize the Plaid SDK
- * @param {Boolean} allowDebit
- * @param {Number} bankAccountID
  */
-function openPlaidBankLogin(allowDebit, bankAccountID) {
-    const params = getPlaidLinkTokenParameters();
-    params.allowDebit = allowDebit;
-    params.bankAccountID = bankAccountID;
+function openPlaidBankLogin(allowDebit: boolean, bankAccountID: number) {
+    // redirect_uri need to be in kebab case convention because its how its passed to backend
```

Comment:
I don't want to be that person but...

```suggestion
    // redirect_uri needs to be in kebab case convention because that's how it's passed to the backend
```